neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -65,10 +67,16 @@ def internalError(message=None):
 class JsonRpcApi(object):
     app = Klein()
     port = None
+    nonce = None
 
     def __init__(self, port):
         self.port = port
 
+        # `nonce` in neo-cli this comes from https://github.com/neo-project/neo/blob/46c635adda6cf61090a67a3bef0f3bb1bc85dd81/neo/Network/LocalNode.cs#L68
+        # In neo-python the closest thing to it is https://github.com/CityOfZion/neo-python/blob/master/neo/Network/NeoNode.py#L34
+        # since we don't get direct access to this NeoNode, we generate a new nonce here
+        self.nonce = random.randint(1294967200, 4294967200)
```

Comment:
FYI; 
`NeoNode.py` is a remote node, this `Nonce` we require should ideally be from the equivalent of `LocalNode.cs` which is `NodeLeader.py`. 

`NodeLeader.Instance().NodeId` (you might even consider to rename `NodeId` to `Nonce`, but that's minor)

Reply:
👍